assets/src/dashboard/components/previewPage.js

```diff
@@ -73,16 +73,16 @@ function PreviewPageController({
 
   useEffect(() => {
     switch (animationState) {
-      case STORY_PAGE_STATE.PLAYING:
+      case STORY_ANIMATION_STATE.PLAYING:
         WAAPIAnimationMethods.play();
         return () => {};
-      case STORY_PAGE_STATE.RESET:
+      case STORY_ANIMATION_STATE.RESET:
         WAAPIAnimationMethods.reset();
         return () => {};
-      case STORY_PAGE_STATE.SCRUBBING:
+      case STORY_ANIMATION_STATE.SCRUBBING:
         WAAPIAnimationMethods.pause();
         return subscribeGlobalTime?.(WAAPIAnimationMethods.setCurrentTime);
-      case STORY_PAGE_STATE.PAUSED:
+      case STORY_ANIMATION_STATE.PAUSED:
         WAAPIAnimationMethods.pause();
         return () => {};
```

Comment:
nit: There is no need to return a clean up function in `useEffect` if no cleanup actually needs to be done, we can just `break` or `return;` instead.